tasks/check-data-keys/tests/test-check-data-keys-fail-missing-advisory-key.yaml

```diff
@@ -31,19 +31,6 @@ spec:
                 "advisory": {
                   "spec": {
                     "cpe": "cpe:/a:example:openstack:el8",
-                    "type": "RHSA",
-                    "issues": {
```

Comment:
why is this being removed?

Reply:
Because it isn't required. I don't expect `type` to ever be in the CRs as we can determine it at advisory creation time. A separate test still has issues in it... I didn't want to have it in every test as it isn't required now

Reply:
Brian wants the type to come from the CRs so made it mandatory again